src/main/java/gregtech/api/capability/impl/FluidTankList.java

```diff
@@ -26,22 +26,33 @@ public class FluidTankList implements IFluidHandler, IMultipleTankHandler, INBTS
     protected IFluidTankProperties[] properties;
     private final boolean allowSameFluidFill;
     private IFluidTankProperties[] fluidTankProperties;
+    private final int hashCode;
```

Comment:
No need to hash `allowSameFluidFill`?

Reply:
I think its fine, as that field is almost always the same value anyway